File: app/src/main/java/com/babestudios/companieshouse/data/model/filinghistory/FilingHistoryList.java
package com.babestudios.companieshouse.data.model.filinghistory;


import com.google.gson.annotations.SerializedName;

import java.util.ArrayList;
import java.util.List;

public class FilingHistoryList {
	@SerializedName("start_index")
	public Integer startIndex;
	@SerializedName("items_per_page")
	public Integer itemsPerPage;
	@SerializedName("items")
	public List<FilingHistoryItem> items = new ArrayList<FilingHistoryItem>();
	@SerializedName("total_count")
	public Integer totalCount;
	@SerializedName("filing_history_status")
	public String filingHistoryStatus;

}

File: data/src/main/java/com/babestudios/companyinfo/data/model/filinghistory/FilingHistoryList.kt
package com.babestudios.companyinfo.data.model.filinghistory


import com.google.gson.annotations.SerializedName

import java.util.ArrayList

class FilingHistoryList {
	@SerializedName("start_index")
	var startIndex: Int? = null
	@SerializedName("items_per_page")
	var itemsPerPage: Int? = null
	@SerializedName("items")
	var items: List<FilingHistoryItem> = ArrayList()
	@SerializedName("total_count")
	var totalCount: Int? = null
	@SerializedName("filing_history_status")
	var filingHistoryStatus: String? = null

}
